pytest_energy_reporter/plugin.py

```diff
@@ -1,37 +1,55 @@
 import pytest
-import logging
 
-energy_metrics = {}
+from .util import print_table_str
+from .measurement import EnergyMeasurement, energy_tester, measure_energy
+
+energy_metrics: list[EnergyMeasurement] = []
+
 
 def pytest_addoption(parser):
-    parser.addoption("--energy-runs", action="store", default=3, type=int,
+    '''Add command line options for the plugin'''
+    parser.addoption("--energy-iterations", action="store", default=3, type=int,
                      help="Number of runs for tests marked as 'energy'")
-    
+    parser.addoption("--save-energy-report", action="store_true",
+                     default=False, help="Save the energy report")
+
+
 def pytest_configure(config):
-    config.addinivalue_line(
-        "markers", "energy: mark test to be run for energy consumption analysis"
-    )
+    '''Configure the energy tester based on the command line options'''
+    if config.getoption("--save-energy-report"):
+        energy_tester.set_save_report(True)
+    else:
+        energy_tester.set_save_report(False)
```

Comment:
Could be shorter:

```python
energy_tester.set_save_report(config.getoption("--save-energy-report", default=False))
```

[Config.getoption Docs](https://docs.pytest.org/en/7.1.x/_modules/_pytest/config.html#Config.getoption)